Model/Transaction.php

```diff
@@ -373,6 +373,12 @@ protected function getParentAmounts($attr, $items, $type = 'order') {
      */
     protected function getProductTaxCode($item, $order)
     {
+        // Check for a PTC saved to the Item
```

Comment:
Does `getTjPtc()` hit the database? It might be cleaner to just assign the result of the method to a variable so we don't need to call it twice.

Reply:
It does not; it uses a local cache on the object.  https://gielberkers.com/magentos-magical-getters-setters-demystified/